shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -348,6 +376,11 @@ - (BOOL)createShell:(NSString*)entrypoint libraryURI:(NSString*)libraryURI {
     FML_LOG(ERROR) << "Could not start a shell FlutterEngine with entrypoint: "
                    << entrypoint.UTF8String;
   } else {
+    [self setupChannels];
```

Comment:
random blast from the past: didn't we do this on init already? Why do we need to do it again?

Reply:
I believe it was because the channels could get reset between there and here

Reply:
how would it though. Looking at the code, it only happens on destroyContext and based on the doc, the object is in an unusable state until it's deallocated. This second setupChannels seems redundant.

Reply:
I can't remember now and I can't see a code path where this happens. I think we can probably kill one of these - I suspec tthe one in the initializer is wrong because actually using a channel before the engine is launched would be an error.

Reply:
SG, thanks for double checking